src/pages/api/identities/[id]/mark-as-primary.ts

```diff
@@ -32,7 +32,7 @@ import { generateAlertMessage } from "@src/utils/generate-alert-message";
 import { getIdentityType } from "@src/utils/get-identity-type";
 import { isMarkingIdentityAsPrimaryAuthorized } from "@src/utils/is-marking-identity-as-primary-authorized";
 
-const handler: Handler = async (request, response) => {
+const markAsPrimary: Handler = async (request, response) => {
```

Comment:
Why don't you put this flow in `/src/pages/api/identities/[id]` ? Since it's a `POST`, we could have merge it with the `GET` and `DELETE`, no ?

Reply:
It's possible and I did it here: https://github.com/fewlinesco/connect-account/pull/560/commits/13b9b2131069a71253a349e099c35578fb298a86